internal/pkg/cli/svc_logs.go

```diff
@@ -322,38 +302,38 @@ func (o *appLogsOpts) outputLogs(logs []*cloudwatchlogs.Event) error {
 	return nil
 }
 
-func (o *appLogsOpts) parseSince() int64 {
+func (o *svcLogsOpts) parseSince() int64 {
 	sinceSec := int64(o.since.Round(time.Second).Seconds())
 	timeNow := time.Now().Add(time.Duration(-sinceSec) * time.Second)
 	return timeNow.Unix() * 1000
 }
 
-func (o *appLogsOpts) parseRFC3339(timeStr string) (int64, error) {
+func (o *svcLogsOpts) parseRFC3339(timeStr string) (int64, error) {
 	startTimeTmp, err := time.Parse(time.RFC3339, timeStr)
 	if err != nil {
 		return 0, fmt.Errorf("reading time value %s: %w", timeStr, err)
 	}
 	return startTimeTmp.Unix() * 1000, nil
 }
 
-// BuildAppLogsCmd builds the command for displaying application logs in a project.
-func BuildAppLogsCmd() *cobra.Command {
-	vars := appLogsVars{
+// BuildSvcLogsCmd builds the command for displaying service logs in an application.
+func BuildSvcLogsCmd() *cobra.Command {
+	vars := svcLogsVars{
 		GlobalOpts: NewGlobalOpts(),
 	}
 	cmd := &cobra.Command{
 		Use:   "logs",
-		Short: "Displays logs of a deployed application.",
+		Short: "Displays logs of a deployed service.",
 
 		Example: `
-  Displays logs of the application "my-app" in environment "test"
-	/code $ ecs-preview app logs -n my-app -e test
+  Displays logs of the service "my-svc" in environment "test"
```

Comment:
nit: can you add a "." at the end of the descriptions
```suggestion
  Displays logs of the service "my-svc" in environment "test".
```